docs_src/getting-started/tools/Ch-CommandLineInterface.md

```diff
@@ -0,0 +1,104 @@
+# Command Line Interface (CLI)
+
+As of the Hanoi release (v1.3.0), EdgeX Foundry provides a command line interface tool.
+
+## What is the EdgeX CLI?
+
+EdgeX-CLI is a command line interface that provides a convenient way for interacting with EdgeX Foundry micro services. It abstracts away all the underling infrastructure and allows the user to easily execute commands. It removes the need to manually track micro services parameters like host and port, as well as knowing the exact request path.
+
+It replaces the need to manually construct complex CURL commands. In particular, it helps with Post/Update requests (example: create a new event, or update a device) because the request body of these requests often include a lot of parameters. EdgeX-CLI provides an "interactive mode" that greatly simplifies the request constructing process. Commands that represent Post/Update requests leverage the interactive mode that opens the user's default editor with preloaded request scaffolding ready to be customized. The interactive mode also works with in-line parameters - allowing in-line values of the CLI command to get loaded in to the request scaffolding. 
```

Comment:
s/constructing/construction
s/mode that opens/mode, opening
s/loaded in to/loaded into